src/Microsoft.TestPlatform.ManagedNames/Extensions/ReflectionExtensions.cs

```diff
@@ -1,20 +1,20 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-namespace Microsoft.VisualStudio.TestPlatform.PlatformAbstractions.Extensions
+namespace Microsoft.VisualStudio.TestPlatform.Extensions
 {
-    using Microsoft.VisualStudio.TestPlatform.CoreUtilities.Resources;
+    using Microsoft.VisualStudio.TestPlatform.Resources;
 
     using System;
 
-    public static partial class ReflectionExtensions
+    internal static partial class ReflectionExtensions
     {
-        private static void AssertSupport<T>(T obj, string methodName, string className) 
+        private static void AssertSupport<T>(T obj, string methodName, string className)
             where T : class
         {
             if (obj == null)
             {
-                throw new NotImplementedException(string.Format(Resources.MethodNotImplementedOnPlatform, className, methodName));
+                throw new NotImplementedException(string.Format(ManagedNameMessages.MethodNotImplementedOnPlatform, className, methodName));
```

Comment:
Better to change to ManagedNameResources to be consistent with other resources.

Reply:
You're right, but this resource is moved from `ObjectModel`, and it's already translated. It's unnecessarily long refactoring for no apparent gain IMO. (All `.resx`, `.xlf` and generated code needs to be updated)